Client/Internal/RecordBatchConverter.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Numerics;
+using Apache.Arrow;
+using InfluxDB3.Client.Write;
+using ArrowArray = Apache.Arrow.Array;
+
+namespace InfluxDB3.Client.Internal;
+
+public static class RecordBatchConverter
+{
+    /// <summary>
+    /// Convert a given row of data from RecordBatch to PointDataValues
+    /// </summary>
+    /// <param name="recordBatch">The RecordBatch to get data from</param>
+    /// <param name="rowNumber">The row number</param>
+    /// <returns>The PointDataValues</returns>
+    public static PointDataValues ConvertToPointDataValue(RecordBatch recordBatch, int rowNumber)
+    {
+        PointDataValues point = new();
+        for (var columnIndex = 0; columnIndex < recordBatch.ColumnCount; columnIndex++)
+        {
+            var schema = recordBatch.Schema.FieldsList[columnIndex];
+            var fullName = schema.Name;
+
+            if (recordBatch.Column(columnIndex) is not ArrowArray array)
+                continue;
+
+            var objectValue = array.GetObjectValue(rowNumber);
+            if (fullName is "measurement" or "iox::measurement" &&
+                objectValue is string value)
+            {
+                point = point.SetMeasurement(value);
+                continue;
+            }
+
+            if (!schema.HasMetadata)
+            {
+                if (fullName == "time" && objectValue is DateTimeOffset timestamp)
+                {
+                    point = point.SetTimestamp(timestamp);
+                }
+                else
+                    // just push as field If you don't know what type is it
+                    point = point.SetField(fullName, objectValue);
+
```

Comment:
`objectValue` can be null, so we need to add a check for it:

```suggestion
                else if (objectValue != null) {
                    // just push as field If you don't know what type is it
                    point = point.SetField(fullName, objectValue);
                }
```

Reply:
Hi @bednar 
Thank you for the review 
I did not check objectValue for null because I can't find any way to write null value to influxdb, so I assume that objectValue can be null
Any suggestions for writing null to influxdb?

Reply:
It’s more about the compile-time `null` safety enforced by the IDE. Since `objectValue` can be null at compile time, and `SetField` does not allow setting a null value, this approach ensures our configuration remains null-safe at runtime.

Reply:
Got your point. Thank you 🙏